src/util/cover-model.html

```diff
@@ -66,6 +66,19 @@
     return (this.stateObj.attributes.supported_features & 128) !== 0;
   });
 
+  addGetter('isTiltOnly', function () {
+    var stateObj = this.stateObj;
+    var tiltClasses = [
+      window.hassUtil.featureClassNames(stateObj, this.tiltFeatureClassNames),
+    ];
+    var tiltClassString = tiltClasses.join(' ').trim();
+    var openClasses = [
+      window.hassUtil.featureClassNames(stateObj, this.openFeatureClassNames),
+    ];
+    var openClassString = openClasses.join(' ').trim();
```

Comment:
This is pretty clever, but I think we should only be doing integer or logical checking here rather than generating these strings, joining them, and looking for an empty string.  I'm thinking something along these lines.

```javascript
var supportsCover = this.supportsOpen || this.supportsClose || this.supportsStop;
var supportsTilt = this.supportsOpenTilt || this.supportsCloseTilt || this.supportsStopTilt;
return supportsTilt && !supportsCover;
```
or
```javascript
var supportsCover = this.stateObj.attributes.supported_features & 11; //where 11 is in a constant somewhere
```